### PR TITLE
S2U-18: Search questions by keywords in question pools

### PR DESCRIPTION
**Pull request by frasese**

### CODE REVIEW

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/QuestionPoolMessages_es.properties [hunk 1]
```diff
@@ -264,3 +264,5 @@ transfer_pool_select_all=Seleccione todas las bater\u00edas
 
 ## S2U-25 Question pools export
 t_exportPool=Exportar
+
+datatables_entities=elementos
```
Comment (jesusmmp) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/QuestionPoolMessages_es.properties:268]: Maybe could we add before the property the corresponding comment

## S2U-18 ...

**Comment by mpellicer** [comment]
Put the commit ID and the PR title in capital letters,

S2U-18 instead of s2u-18

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/QuestionPoolMessages_es.properties (hunk 1 above)
Comment (mpellicer) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/QuestionPoolMessages_es.properties:268]: Yes please, 

Add ## S2U-18 to all the files.

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/model/DataTableConfig.java [hunk 2]
```diff
@@ -63,6 +65,8 @@ public String getJson() {
     }
 
     public static DataTableConfigBuilder builderWithDefaults() {
+    	
+    	System.out.println("RESOURCE_BUNDLE locale: "+RESOURCE_BUNDLE.getLocale());
```
Comment (mpellicer) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/model/DataTableConfig.java:69]: log.debug.

File: samigo/samigo-app/src/webapp/jsf/questionpool/questionTreeTable.jsp [hunk 3]
```diff
@@ -20,10 +20,13 @@
 --%>
 -->
 
+<script>includeWebjarLibrary('datatables');</script>
+<script type="text/javascript" src="/samigo-app/js/dataTables.js"></script>
```
Comment (mpellicer) [inline samigo/samigo-app/src/webapp/jsf/questionpool/questionTreeTable.jsp:24]: remove type="text/javascript"

**Review by mpellicer — APPROVED** [review]
Looks AMAAAAZING, you're a monster Paco.

Only minor comments.

**Review by mpellicer — APPROVED** [review]

**Pull request merged**